admin/authentication/nca-tools-authentication-strategy.js

```diff
@@ -24,9 +25,16 @@ module.exports = async function (req, done) {
     loginMethod: 'nca-tools'
   }
 
+  let ncaPublicKey, mtcPrivateKey
+  try {
+    ncaPublicKey = await certificateService.getNcaPublicKey()
+    mtcPrivateKey = await certificateService.getMtcPrivateKey()
+  } catch (error) {
+    winston.error('unable to retrieve certificates:', error)
+    return done(null, false)
```

Comment:
This is probably a slightly wrong return.  `done(null, false)` means authentication failed.  Returning done(error, null) means that authentication had an error, which is the case here.

Other lines are also throwing an error, and if that works fine that's great too, but I would need to double-check that.